## Estimating the Effect of Membership in Farmer Organizations on Household Income

### Introduction:
This report aims to estimate the effect of membership in a farmer organization on household income using statistical/econometric methods. We will describe the model specification, the methodology used for estimation, present the regression results in a table, interpret these results, and briefly discuss the extent to which assumptions for unbiased estimates of the causal effect are fulfilled.

### Model Specification:
Using a linear regression model to estimate the effect of membership in a farmer organization on household income. The model is specified as follows:  
```
Income = β0 + β1 ● Farmer_Organization_Membership + ϵ
```

- Income: represents household income.
Farmer_Organization_Membership: is a binary variable (0 for non-members, 1 for members).  
- β0 : is the intercept, representing the expected income for non-members.
- β1 : is the coefficient of interest, indicating the difference in income between members and non-members.
- ϵ: is the error term.

In [1]:
import pandas as pd
import statsmodels.api as sm
data = pd.read_csv("qn1_corrected_data_exercise.csv")

# Binary variable for Farmer Organization Membership (1 for members, 0 for non-members)
data['Farmer_Organization_Membership'] = data['fo']

# Dependent (y) and independent (X) variables
X = data['Farmer_Organization_Membership']  # Independent variable
X = sm.add_constant(X)  # Add a constant (intercept)
y = data['income']  # Dependent variable


### Model Estimation:

In [2]:
# Fit the linear regression model
model = sm.OLS(y, X).fit()

### Regression Results and Interpretation

In [3]:
# Get the regression results
results = model.summary()

# Print the results
print(results)


                            OLS Regression Results                            
Dep. Variable:                 income   R-squared:                       0.057
Model:                            OLS   Adj. R-squared:                  0.055
Method:                 Least Squares   F-statistic:                     29.10
Date:                Sun, 01 Oct 2023   Prob (F-statistic):           1.08e-07
Time:                        23:23:54   Log-Likelihood:                -3752.1
No. Observations:                 480   AIC:                             7508.
Df Residuals:                     478   BIC:                             7517.
Df Model:                           1                                         
Covariance Type:            nonrobust                                         
                                     coef    std err          t      P>|t|      [0.025      0.975]
--------------------------------------------------------------------------------------------------
const       

### Interpretation of Regression Results:
The coefficient for **Farmer_Organization_Membership** is 337.6592, with a *p-value* of 0.000, indicating that 
``` membership in a farmer organization has a statistically significant positive effect on household income ```.  
The constant (intercept) represents the expected income for non-members of farmer organizations, which is 575.5408.  
The R-squared value is 0.057, 
``` suggesting that the model explains approximately 5.7% of the variance in household income. ```

#### This analysis provides insights into the effect of membership in farmer organizations on household income, suggesting a positive association between membership and income levels.


## Conclusion

## Research Question
Our study aimed to investigate whether joining a farmer group has an impact on household income. We wanted to understand if being part of such a group could lead to higher earnings.

## Methodology
To answer our question, we used a statistical technique called linear regression. It helps us see if there's a connection between two things. In our case, we were interested in two factors for each person:
- Whether they are a member of a farmer group (yes or no).
- How much money they earn.

## Key Findings
Our analysis revealed that individuals who are part of a farmer group tend to have higher household incomes. On average, they earn more than those who are not part of such a group.

## Interpretation
This suggests that joining a farmer group might have a positive effect on household income. This could be due to various factors, such as gaining access to knowledge and resources that help improve farming practices and ultimately increase earnings.